mix.exs

```diff
@@ -8,7 +8,7 @@ defmodule Postgrex.Mixfile do
     [
       app: :postgrex,
       version: @version,
-      elixir: "~> 1.11",
+      elixir: "~> 1.17",
```

Comment:
To be clear, we shouldn't change this requirement. Users running on an Elixir version with a bug should upgrade their Elixir, we shouldn't force people on 1.15 (still supported) to update because of that. :) We tend to be conservative with those. :)

```suggestion
      elixir: "~> 1.11",
```